### ISSUE TITLE
Autochecker script bugs

### ISSUE THREAD

**Issue by robdy** [get-itips/msshells#25]
- [x] Script will not work as expected in very rare case when current version is the last one on the page and there's prerelease version on next page
- [x] Script should utilize `IsAbsoluteLatestVersion` parameters to detect newest preview version, such as:
```
Where-Object {$_.properties.isPrerelease.'#text' -eq 'true' -and $_.properties.IsAbsoluteLatestVersion.'#text' -eq 'true'}
```

**Comment by robdy** [comment]
Both fixed in #100.

Now the only thing to consider is whether to run `Find-Module` once and filter in the script (more fragile to errors) or run twice (might take longer). I'll be testing it with another PR.

**Comment by get-itips** [comment]
Good job @robdy !
